<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilePage</class>
 <widget class="QDialog" name="ProfilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
    background-color: #ffffff;
}
QLabel { 
    color: #2c3e50; 
    font-size: 14px; 
    font-weight: bold;
    margin-bottom: 5px;
}
QLineEdit { 
    background-color: #f8f9fa;
    border: 2px solid #e9ecef;
    border-radius: 8px;
    padding: 12px;
    font-size: 14px;
    color: #2c3e50;
    margin-bottom: 5px;
}
QLineEdit:focus {
    border: 2px solid #4a90e2;
    background-color: white;
}
QPushButton {
    background-color: #4a90e2;
    color: white;
    border: none;
    border-radius: 8px;
    padding: 12px;
    font-size: 16px;
    font-weight: bold;
    min-height: 45px;
}
QPushButton:hover {
    background-color: #357abd;
}
QPushButton:pressed {
    background-color: #2d6da3;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>48</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #2c3e50; margin-bottom: 40px;</string>
     </property>
     <property name="text">
      <string>Change Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="oldPasswordLabel">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="oldPasswordLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter your current password</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="newPasswordLabel">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>New Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="newPasswordLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter your new password</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="confirmPasswordLabel">
     <property name="font">
      <font>
       <pointsize>28</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Confirm New Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="confirmPasswordLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Confirm your new password</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="changePasswordButton">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="text">
      <string>Change Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #D32F2F; color: white; border: 2px solid #B71C1C; border-radius: 15px; padding: 10px; font-weight: bold; font-size: 22px;</string>
     </property>
     <property name="text">
      <string>Back to Home</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="styleSheet">
      <string notr="true">color: #e74c3c; 
font-weight: normal; 
margin-top: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
